### ISSUE TITLE
State variable is not working on value prop

### ISSUE THREAD

**Issue by jitanshu** [mauriciosoares01/react-native-custom-switches#2]
Everything is working great but the value prop is not changing values as per state variable

**Comment by mauriciosoares01** [comment]
Yes, you are right. But that was the only way that I found to manage the animations. When I use an internal state for the component status, the animation isn't triggered. I will stay thinking in a better implementation, and open to sugestions. Thanks for the feedback.
